javascript/index.js

```diff
@@ -0,0 +1,87 @@
+function Book(bookTitle, bookAuthor) {
+  this.bookTitle = bookTitle;
+  this.bookAuthor = bookAuthor;
+}
+
+function DisplayBook() {
+  this.books = [];
+}
+
+DisplayBook.prototype.addBook = function (book) {
+  this.books.push(book);
+  this.displayBook(book);
+  this.saveBooksToLocalStorage();
+};
+
+DisplayBook.prototype.displayBook = function (book) {
+  const bookList = document.getElementById('bookList');
+  const div = document.createElement('div');
+  div.classList.add('book');
+
+  const title = document.createElement('div');
+  title.textContent = `Title: ${book.bookTitle}`;
+  div.appendChild(title);
+
+  const author = document.createElement('div');
+  author.textContent = `Author: ${book.bookAuthor}`;
+  div.appendChild(author);
+
+  const deleteBtn = document.createElement('button');
+  deleteBtn.textContent = 'Delete';
+  deleteBtn.classList.add('delete-btn');
+  div.appendChild(deleteBtn);
+
+  bookList.appendChild(div);
+};
+
+DisplayBook.prototype.clearForm = function () {
+  document.getElementById('title').value = '';
+  document.getElementById('author').value = '';
+};
+
+DisplayBook.prototype.deleteBook = function (bookTitle, bookAuthor) {
+  // eslint-disable-next-line max-len
+  this.books = this.books.filter((book) => !(book.bookTitle === bookTitle && book.bookAuthor === bookAuthor));
+  this.saveBooksToLocalStorage();
+};
+
```

Comment:
- [ ] Please make sure all books  if  are being stored in the `local storage` currently if I add the second book the previous one disappears
<img width="1388" alt="Screenshot 2023-07-05 at 10 52 06" src="https://github.com/Obote/AwesomeBooks/assets/44745855/549ae763-7cbc-49a5-a86e-f8eb0e8341e5">